docs/docs/Configuration/configuration-my-collection.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
I see that you created a new file instead of using the previous file. My understanding is that we don't have a redirect mechanism in place in Docusaurus yet, so deleting the previous file will cause a 404 for anyone who had it bookmarked or if it's linked to from somewhere else. What's your plan to prevent 404s?

Reply:
Now that i merged the redirects PR i can do this
https://github.com/langflow-ai/langflow/pull/4425/files#diff-28742c737e523f302e6de471b7fc27284dc8cf720be639e6afe4c17a550cd654